extensions/kubernetes-service-binding/runtime/src/main/java/io/quarkus/kubernetes/service/binding/runtime/ServiceBinding.java

```diff
@@ -63,6 +63,7 @@ public ServiceBinding(Path bindingDirectory) {
 
     private static Map<String, String> getFilenameToContentMap(Path directory) {
         if (!Files.exists(directory) || !Files.isDirectory(directory)) {
+            log.debug("File '" + directory + "' is not a proper service binding directory so it will skipped");
```

Comment:
Is debug right here? It seems to be a deployment issue, and a warning might be more appropriate. The "ops" team would need to be aware of such an issue. no?

Reply:
Good point